i18n/es.po

```diff
@@ -785,20 +785,9 @@ msgstr "Seleccione las unidades organizativas para añadir a la plantilla"
 msgid "User does not have any capture organisations units"
 msgstr "El usuario no tiene unidades organizativas asociadas"
 
-msgid "Advanced template properties"
-msgstr "Propiedades avanzadas de la plantilla"
-
-msgid "Language"
-msgstr "Idioma"
-
-msgid "Split data entry tabs by section"
-msgstr ""
-
-msgid "Theme"
-msgstr "Tema"
-
-msgid "<No value>"
-msgstr "<Sin valor>"
+#, fuzzy
+msgid "Populate"
+msgstr "Plantilla"
```

Comment:
We typically check that PO files do not contain "fuzzy" tags (gettext means: "I think this is the translation for a new literal I found, which is very similar to another existing, but I am not sure"). We review if correct and the remove it.

BUT, as all the po files already contain fuzzy tags (a lot), you may leave this task to another "cleaning" PR, not in this feature. Confirm with your PM.